sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -727,5 +761,53 @@ interface WithLocalAccounts {
              */
             Update disableLocalAccounts();
         }
+
+        /**
+         * The stage of the Kubernetes cluster update allowing to specify a tier for the managed cluster SKU.
+         */
+        interface WithPricingTier {
+
+            /**
+             * Specifies the tier of the managed cluster SKU is free.
+             *
+             * @return the next stage
+             */
+            Update withFreeTier();
+
+            /**
+             * Specifies the tier of the managed cluster SKU is standard.
+             *
+             * @return the next stage
+             */
+            Update withStandardTier();
+
+            /**
+             * Specifies the tier of the managed cluster SKU is premium.
+             *
+             * @return the next stage
+             */
+            Update withPremiumTier();
+        }
+
+        /**
+         * The stage of the Kubernetes cluster update allowing to specify orchestration type.
+         */
+        interface WithVersion {
+            /**
+             * Specifies the version for the Kubernetes cluster.
+             * Could retrieve from {@link KubernetesClusters#listOrchestrators(Region, ContainerServiceResourceTypes)}
+             *
+             * @param kubernetesVersion the kubernetes version
+             * @return the next stage
+             */
+            Update withVersion(String kubernetesVersion);
+
+            /**
+             * Uses the default version for the Kubernetes cluster.
+             *
+             * @return the next stage
+             */
+            Update withDefaultVersion();
```

Comment:
Could cluster be updated to default version?

Reply:
After testing, it cannot be updated from the specified version to the default version, so `withDefaultVersion` is removed from `UpdateStages`.